lib/launcher.js

```diff
@@ -284,11 +290,21 @@ class Launcher {
      * Write .npmrc file
      */
     async writeNPMRCFile () {
-        if (this.snapshot.settings?.palette?.npmrc) {
-            await fs.writeFile(this.files.npmrc, this.snapshot.settings.palette.npmrc)
-        } else {
-            if (existsSync(this.files.npmrc)) {
-                await fs.rm(this.files.npmrc)
+        if (this.project) {
+            if (this.snapshot.settings?.palette?.npmrc) {
+                await fs.writeFile(this.files.npmrc, this.snapshot.settings.palette.npmrc)
+            } else {
+                if (existsSync(this.files.npmrc)) {
+                    await fs.rm(this.files.npmrc)
+                }
+            }
+        } else if (this.application) {
+            if (this.settings.palette?.npmrc) {
+                await fs.writeFile(this.files.npmrc, this.settings.palette.npmrc)
+            } else {
+                if (existsSync(this.files.npmrc)) {
+                    await fs.rm(this.files.npmrc)
+                }
```

Comment:
Unsure why snapshot is not considered when application bound?

I may need to pull and test locally to do this pair of PRs any justice.

I have another PR to review and my own PR to adjust all within the next 1h30m - i may not get this done today (sorry Ben)

Reply:
Hi @Steve-Mcl I'm trying to get the Forge app side of this fixed up today, can you finish looking at this bit as well please